File: cpp/angka.cpp

#include <iostream>
using namespace std;

int main() {
    int a, b;
    cout << "Masukkan sebuah angka: ";
    cin >> a;
    cout << "Masukkan angka kedua: ";
    cin >> b;

    if (a > b) {
        cout << a << " adalah angka genap." << b << endl;
    } else if (a<b)
    {
        cout << a << " lebih kecil dari " << b << endl;
    }
    else {
        cout << " Keduanya sama besar" << endl;
        cout << " Apakah kdeuanya sama?"<< (a==b);
    }
}

File: Java/Angka.java

import  java.util.Scanner;
public class Angka {
    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);
        System.out.print("Masukkan sebuah angka: ");
        int a = sc.nextInt();
        System.out.print("Angka yang dimasukkan: " + a);
        int b = sc.nextInt();

        if (a>b)
            System.out.println(a + "Lebih besar dari: " + b);
        else if (a < b)
            System.out.println(a + "Lebih kecil dari: " + b);
        else
            System.out.println("Keduanya sama besar: ");
            System.out.println("Apakah keduanya sama?" + (a == b));
    }
}
